pobieracz_danych_gugik.py

```diff
@@ -1666,6 +1632,15 @@ def downloadAerotriangulacjiForSinglePoint(self, point):
 
         self.filterAerotriangulacjaListAndRunTask(aerotriangulacjaList)
 
+    def downloadMesh3dForSinglePoint(self, point):
+        point1992 = utils.pointTo2180(
+            point=point,
+            sourceCrs=QgsProject.instance().crs(),
+            project=QgsProject.instance()
```

Comment:
Tu przydałoby się skorzystać ze zmiennych globalnych self.crs i self.project (widzę, że takowych nie ma, to proszę utwórz je, bo crs to najlepiej jakby było w funkcji run, a project chyba w inicie (ale jak uważasz inaczej to śmiało))